src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1058,6 +1060,47 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		this._terminalAltBufferActiveContextKey.set(!!(this.xterm && this.xterm.raw.buffer.active === this.xterm.raw.buffer.alternate));
 	}
 
+	private async _shouldPasteText(text: string): Promise<boolean> {
+		let textForLines = text.split(/(\r\n|\n|\r)/);
```

Comment:
Should be equivalent:

```suggestion
		let textForLines = text.split(/(\r?\n)/);
```